commons-rest/src/main/java/com/mule/connectors/commons/rest/builder/RequestBuilder.java

```diff
@@ -6,6 +6,7 @@
 import com.mule.connectors.commons.rest.builder.handler.ResponseHandler;
 import com.mule.connectors.commons.rest.builder.strategy.*;
 import org.apache.commons.lang3.StringUtils;
+import org.glassfish.jersey.internal.util.Base64;
```

Comment:
Another good idea could be this: http://stackoverflow.com/questions/14413169/which-java-library-provides-base64-encoding-decoding
Seems like Java 6 and 7 have OOB support for Base64.